### ISSUE TITLE
Display available rooms for Guest

### ISSUE THREAD

**Issue by jaypeasee** [jaypeasee/overlook-hotel#23]
### As a user, when I pick a date in the calendar on the nav:
1. I should be able to see all available bookings for that day. If there are none.
1. I should be able to pick an available room.
1. I should see the main section title update.
1. If there are no availabilities, the Main section should show that.
1. If I pick a date in the past, the Main section should show that.
